python/transformations/Hugging-Face-Model/main.py

```diff
@@ -1,48 +1,50 @@
-from quixstreaming import QuixStreamingClient, StreamEndType, StreamReader
-from quixstreaming.app import App
+import quixstreams as qx
 from hugging_face_model import HuggingFaceModel
 from transformers import pipeline
 import os
 
-# Quix injects credentials automatically to the client. Alternatively, you can always pass an SDK token manually as an argument.
-client = QuixStreamingClient()
+# Quix injects credentials automatically to the client.
+# Alternatively, you can always pass an SDK token manually as an argument.
+client = qx.QuixStreamingClient()
 
 # Download the Hugging Face model (list of available models here: https://huggingface.co/models)
+# suggested default is distilbert-base-uncased-finetuned-sst-2-english
 model_name = os.environ["HuggingFaceModel"]
 print("Downloading {0} model...".format(model_name))
 model_pipeline = pipeline(model=model_name)
 
 print("Opening input and output topics")
 # Change consumer group to a different constant if you want to run model locally.
-input_topic = client.open_input_topic(os.environ["input"], "default-consumer-group")
-output_topic = client.open_output_topic(os.environ["output"])
+consumer_topic = client.get_topic_consumer(os.environ["input"], "default-consumer-group")
+producer_topic = client.get_topic_producer(os.environ["output"])
+
 
 # Callback called for each incoming stream
-def read_stream(input_stream: StreamReader):
+def read_stream(stream_consumer: qx.StreamConsumer):
```

Comment:
Minor comment: should this be "consume_stream"?

Reply:
Hmm.. the object is StreamConsumer so i kept it consistent with that.